src/themes/default/layouts/Default.vue

```diff
@@ -88,7 +88,7 @@ export default {
       return this.$store.dispatch('category/list', { level: config.entities.category.categoriesDynamicPrefetch && config.entities.category.categoriesDynamicPrefetchLevel >= 0 ? config.entities.category.categoriesDynamicPrefetchLevel : null, includeFields: config.entities.optimize && isServer ? config.entities.category.includeFields : null, skipCache: isServer })
     }
   },
-  serverPrefetch () {
+  created () {
```

Comment:
I guess it should be on condition - in SSR mode still fetching should be called from `serverPrefetch`

Reply:
I think we should keep `serverPrefetch` because ssr mode is our default. If someone wants to use spa mode then is should be described as exception. We can't trigger `serverPrefetch`, because its done by vue. And I didn't found a way to force it on csr.

Reply:
we need to keep both + have a config flag to distingiush